backend/src/pong/game/entities/game.entity.ts

```diff
@@ -0,0 +1,46 @@
+import {
+	Column,
+	CreateDateColumn,
+	Entity,
+	JoinColumn,
+	PrimaryGeneratedColumn,
+	UpdateDateColumn,
+} from 'typeorm';
+
+@Entity('games')
+export class Game {
+	@PrimaryGeneratedColumn()
+	id: number;
+
+	@Column({ default: 0 })
+	score_player_one: number;
+
+	@Column({ default: 0 })
+	score_player_two: number;
+
+	// for now empty customization
+	@Column({ default: null })
+	customization: string;
+
+	@Column({ default: true })
+	is_active: boolean;
+
+	@CreateDateColumn()
+	created_at: Date;
+
+	@UpdateDateColumn()
+	updated_at: Date;
+
+	// to be joined later on
+	// @ManyToOne(() => User,  user => user.id)
+	// @JoinColumn({})
+	// player_one: User;
+	@Column()
+	player_one: number;
+
+	// @ManyToOne(() => User, user => user.id)
+	// @JoinColumn({})
+	// player_two: User;
+	@Column()
+	player_two: number;
```

Comment:
Yes indeed, I added it in the create and update methods, but then removed it because we are still deciding on how to link them, but it was a manual check so this is a bit nicer!

Reply:
I just tried my example btw, and it works. It very nicely adds a direct constraint on the db level :)

Reply:
Better, muuuuch better!